code/src/App.js

```diff
@@ -1,12 +1,31 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from "./Components/Album.js"
+import "./App.css"
+
 
 console.log(data)
 
+
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+
+    <div className="cards">
+
+      {data.albums.items.map((item) => (
+        <Album
+          key={item.id}
+          image={item.images[1].url}
+          title={item.name}
+          artist={item.artists[0].name}
+          artistLink={item.artists[0].uri}
```

Comment:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of artist and artistLink go for artists={album.artists}